### PR TITLE
[mesheryctl] Organizing the start of user guides

### PR DESCRIPTION
**Pull request by leecalcote**
**Description**

Creating the separation between `mesheryctl` References and `mesheryctl` Guides.

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
Lee, I'm not sure how to put this, but you do amazing work.

**Comment by leecalcote** [comment]
Thank you, Lee. I always appreciate your timely PR reviews, too.

**Comment by leecalcote** [comment]
What's that? Oh, yes, I agree. This PR is ready to merge.

**Comment by leecalcote** [comment]
Ok, great. Yes, it LGTM.

**Comment by alphaX86** [comment]
Uh oh, seems I missed all that! lol. Sorry for late response, but I got my univ work finished just now

File: docs/pages/guides/mesheryctl/configuring-autocompletion-for-mesheryctl.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+title: Configuring Autocompletion for `mesheryctl`
+permalink: guides/mesheryctl/configuring-autocompletion-for-mesheryctl
+language: en
+type: Guides
+category: mesheryctl
+---
+# Configuring Autocompletion for `mesheryctl`
+
+If you would like to have `mesheryctl` commands automatically completed for use as you use `mesheryctl`, then use the following instructions to configure automatic completion within your environment.
+
```
Comment (alphaX86) [inline docs/pages/guides/mesheryctl/configuring-autocompletion-for-mesheryctl.md:12]: For this line alone, the page prints like this....

Comment (alphaX86) [reply docs/pages/guides/mesheryctl/configuring-autocompletion-for-mesheryctl.md:12]: image

**Review by alphaX86 — COMMENTED** [review]
All other changes are great! But only that file needs formatting though

**Comment by leecalcote** [comment]
> Uh oh, seems I missed all that! lol. Sorry for late response, but I got my univ work finished just now

Ha! No problem. I was just having a bit of fun... with myself, I suppose. 🙂

File: docs/pages/guides/mesheryctl/configuring-autocompletion-for-mesheryctl.md (hunk 1 above)
Comment (leecalcote) [reply docs/pages/guides/mesheryctl/configuring-autocompletion-for-mesheryctl.md:12]: Sounds like the frontmatter on this page could use: `display-title: "false"`.

Comment (alphaX86) [reply docs/pages/guides/mesheryctl/configuring-autocompletion-for-mesheryctl.md:12]: Yeah, need to look on that

**Comment by leecalcote** [comment]
Duplicate page heading removed.

**Comment by leecalcote** [comment]
Thank you, @alphaX86

**Pull request merged**